pkg/kubectl/cmd/util/helpers.go

```diff
@@ -408,16 +408,6 @@ func AddValidateOptionFlags(cmd *cobra.Command, options *ValidateOptions) {
 
 func AddOpenAPIFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("openapi-validation", false, "If true, use openapi rather than swagger for validation")
-	cmd.Flags().String("schema-cache-dir",
```

Comment:
This code is not removing the `schema-cache-dir` flag, it's just not using it for openapi caching.
I'd like to have a separate directory/flag for "manual" caching (i.e. `schema-cache-dir`), and http caching (i.e. `cachedir), the later becoming the standard in the future. Opinions @kubernetes/sig-cli-misc ?